rasa/nlu/components.py

```diff
@@ -58,7 +59,7 @@ def validate_requirements(component_names: List[Optional[Text]]) -> None:
     failed_imports = {}
     for component_name in component_names:
         if component_name is None:
-            raise InvalidConfigError(
+            raise InvalidConfigException(
```

Comment:
can you please adapt the docstring for this function? The `Raises` mentions `InvalidConfigError`